doc/smartdashboard.rst

```diff
@@ -0,0 +1,7 @@
+
+**********
```

Comment:
The decorator `**********` should be at least as long as the text which is enclosing

Reply:
+1 - fixed!